css-ui-4/Overview.bs

```diff
@@ -2193,11 +2200,82 @@ so this specification needs to say something about this.
 For compatibility with legacy content, UAs must also support <dfn property export>-webkit-appearance</dfn>
 as a [=legacy name alias=] of 'appearance'.
 
+
+<h4 id="appearance-disabling-properties">
+Properties Disabling Native Appearance</h4>
+
+Certain properties, when declared in the [=Author Origin=] or [=Animation Origin=],
+will disable the [=native appearance=] of certain [=widgets=].
+Specifically,
+if all the declarations of the [=Author Origin=] and [=Animation Origin=]
+pertaining to such a [=widget=] are cascaded
+and a [=cascaded value=] exists--
+and is not ''revert'' or ''revert-layer''--
```

Comment:
What about

```css
@layer {
  .widget { border-left-style: solid }
}
@layer {
  .widget { border-left-style: revert-layer }
}
```

The cascaded value is `revert-layer`, but the native appearance should be disabled anyways since the specified value will be `border-left-style: solid`.

Reply:
filed as https://github.com/w3c/csswg-drafts/issues/7252